Packages/com.unity.inputsystem/InputSystem/Utilities/ReadOnlyArray.cs

```diff
@@ -123,38 +134,46 @@ IEnumerator IEnumerable.GetEnumerator()
             }
         }
 
-        internal class Enumerator<T> : IEnumerator<T>
+        /// <summary>
+        /// <see cref="ReadOnlyArray{TValue}"/>
+        /// Enumerates the elements of a read-only array.
+        /// </summary>
+        public struct Enumerator : IEnumerator<TValue>
         {
-            private readonly T[] m_Array;
+            private readonly TValue[] m_Array;
             private readonly int m_IndexStart;
             private readonly int m_IndexEnd;
             private int m_Index;
 
-            public Enumerator(T[] array, int index, int length)
+            internal Enumerator(TValue[] array, int index, int length)
             {
                 m_Array = array;
                 m_IndexStart = index - 1; // First call to MoveNext() moves us to first valid index.
                 m_IndexEnd = index + length;
                 m_Index = m_IndexStart;
             }
 
+            /// <inheritdoc/>>
             public void Dispose()
             {
             }
 
+            /// <inheritdoc/>>
```

Comment:
Double-closing tag `>>`. I don't think these actually work here. AFAIK our doc generation can only `<inheritdoc/>` from docs it has in the input set.

Reply:
Oops, I've no idea how that happened, probably Rider that placed full open/close tags and I didn't notice! Hrm, I should have fixed it before this PR was merged. Do you mind fixing it in a subsequent PR maybe?